src/components/flytegraph/Layout.tsx

```diff
@@ -29,7 +29,7 @@ export class Layout<T> extends React.Component<LayoutProps<T>> {
     // input props actually change
     layoutGraph = memoizeOne<typeof layoutGraph>(
         (data, config) => layoutGraph(data, config),
-        layoutArgsAreEqual
+        layoutArgsAreEqual as EqualityFn
```

Comment:
Is there a legitimate issue that we're getting around by using `as` here?

Reply:
The compiler was having trouble reconciling the type of `layoutArgsAreEqual` with the type `EqualityFn`. The latter accepts `any[]` as its input, the former accepts `[GraphInputNode[], GraphConfig]`. Those _should_ be compatible, but I get `Type 'any[]' is missing the following properties from type '[GraphInputNode[], GraphConfig]': 0, 1`. I'm giving it a little nudge to help.